forge-gui/res/cardsfolder/v/vodalian_war_machine.txt

```diff
@@ -3,9 +3,11 @@ ManaCost:1 U U
 Types:Creature Wall
 PT:0/4
 K:Defender
-A:AB$ Effect | Cost$ tapXType<1/Merfolk> | RememberCostCards$ True | StaticAbilities$ CanAttack | SpellDescription$ CARDNAME can attack this turn as though it didn't have defender.
+T:Mode$ AbilityCast | ValidCard$ Card.Self | Static$ True | Execute$ TrigStore
+SVar:TrigStore:DB$ Pump | RememberObjects$ TriggeredSpellAbility>TappedCards
```

Comment:
shouldn't it also forget at end of turn?

Say you activate some abilities on your turn, then on opponent's turn they kill the VWM... in this case it would destroy the tapped merfolk even though they weren't tapped "this turn", right?

Reply:
it has static phase cleanup trigger (still problematic but it is what it is)

Reply:
heh, I just missed that in the soup